# <mayhem_program::pump_amm::internal::__cpi_client_accounts_buy::Buy as anchor_lang::ToAccountMetas>::to_account_metas  (yours 5040 B at .text+0x6a0a8; original 5040 B at .text+0x78250)
# distance 24 instructions; evidence: called-by mayhem_program::pump_amm::cpi::buy#a580; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -15,7 +15,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r1, [r10-48]
 ldxdw r2, [r10-8]
 stxdw [r1+24], r2
@@ -41,7 +41,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -67,7 +67,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -93,7 +93,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -119,7 +119,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -145,7 +145,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -171,7 +171,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -197,7 +197,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -223,7 +223,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -249,7 +249,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -275,7 +275,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -301,7 +301,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -327,7 +327,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -353,7 +353,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -379,7 +379,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -405,7 +405,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -431,7 +431,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -457,7 +457,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -483,7 +483,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -509,7 +509,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -520,7 +520,7 @@
 stxdw [r1+654], r3
 ldxdw r3, [r10-32]
 stxdw [r1+646], r3
-sth [r1+678], 256
+sth [r1+678], 0
 stdw [r10-40], 20
 ldxdw r3, [r7+960]
 ldxdw r4, [r3+24]
@@ -535,7 +535,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -561,7 +561,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r2, [r10-56]
 ldxdw r1, [r10-48]
 ldxdw r3, [r10-8]
@@ -587,7 +587,7 @@
 mov64 r1, r10
 add64 r1, -56
 lddw r2, panic_location(programs/mayhem-program/src/lib.rs:20:1)
-call alloc::raw_vec::RawVec<T,A>::grow_one#30d4
+call alloc::raw_vec::RawVec<T,A>::grow_one#f47e
 ldxdw r1, [r10-48]
 ldxdw r2, [r10-8]
 stxdw [r1+772], r2
